packages/x-charts/src/ChartsAxisHighlight/ChartsAxisHighlight.tsx

```diff
@@ -30,8 +30,12 @@ function ChartsAxisHighlight(props: ChartsAxisHighlightProps) {
   const classes = useUtilityClasses();
   return (
     <React.Fragment>
-      {xAxisHighlight && <ChartsXHighlight type={xAxisHighlight} classes={classes} />}
-      {yAxisHighlight && <ChartsYHighlight type={yAxisHighlight} classes={classes} />}
+      {xAxisHighlight && xAxisHighlight !== 'none' && (
+        <ChartsXHighlight type={xAxisHighlight} classes={classes} />
+      )}
+      {yAxisHighlight && yAxisHighlight !== 'none' && (
+        <ChartsYHighlight type={yAxisHighlight} classes={classes} />
+      )}
```

Comment:
While debugging, I noticed `'none'` renders the component which is bad because we call the x/y-axis highlight selector to simply return `null` at the end